### PR TITLE
AblyRealtime pod

### PR DESCRIPTION
**Pull request by ricardopereira**
```
 -> AblyRealtime (0.8.0-beta.2)
    - ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.
    - NOTE  | xcodebuild:  AblyRealtime/Source/ARTAuthOptions.h:10:9: fatal error: 'Ably/ARTTypes.h' file not found
```

This PR fixes the validation issue but we need to use it as `#import <AblyRealtime/Ably.h>` when using the Pod with Objective-C. I can't import it on Swift. It says _No such module_. I tried with many variations but it isn't working. It doesn't make sense because the target name is correct.

### CODE REVIEW

**Comment by ricardopereira** [comment]
@mattheworiordan Can't we stick with the decentralised way?

**Comment by mattheworiordan** [comment]
> Can't we stick with the decentralised way?

Well no, the point is that this repo is meant to be a Cocoapod, so I'd like it to be registered like all other Cocoapods out there please.

Does this PR not get it working then?

**Comment by mattheworiordan** [comment]
I tried to push with this and still get:

```
$ pod trunk push AblyRealtime.podspec
Updating spec repo `master`

CocoaPods 1.0.0.beta.6 is available.
To update use: `sudo gem install cocoapods --pre`
[!] This is a test version we'd love you to try.

For more information see http://blog.cocoapods.org
and the CHANGELOG for this version http://git.io/BaH8pQ.

Validating podspec
 -> AblyRealtime (0.8.0-beta.2)
    - ERROR | [iOS] xcodebuild: Returned an unsuccessful exit code. You can use `--verbose` for more information.
    - NOTE  | xcodebuild:  AblyRealtime/Source/ARTAuthOptions.h:10:9: fatal error: 'Ably/ARTTypes.h' file not found

[!] The podspec does not validate.
```

File: Spec/AblySpec-Bridging-Header.h [hunk 1]
```diff
@@ -2,7 +2,7 @@
 //  Use this file to import your target's public headers that you would like to expose to Swift.
 //
 
-#include <ably/ably.h>
+#include "Ably.h"
```
Comment (tcard) [inline Spec/AblySpec-Bridging-Header.h]: Why not `"AblyRealtime/Ably.h"`? It had the folder prefix before.

Comment (ricardopereira) [inline Spec/AblySpec-Bridging-Header.h]: But I prefer to remove the prefix dependency. It helps when you change de Pod name.

**Comment by ricardopereira** [comment]
It's working now.

**Objective-C**
```objective-c
#import <AblyRealtime/AblyRealtime.h>
```

**Swift**
```swift
import AblyRealtime
```

**Comment by ricardopereira** [comment]
```
** BUILD SUCCEEDED **

 -> AblyRealtime (0.8.0-beta.3)

Analyzed 1 podspec.

AblyRealtime.podspec passed validation.
```

**Comment by mattheworiordan** [comment]
It built for me too, thanks

**Pull request merged**